app/ide-desktop/lib/dashboard/src/layouts/PageSwitcher.tsx

```diff
@@ -73,33 +85,65 @@ export default function PageSwitcher(props: PageSwitcherProps) {
     }
   }, [isEditorDisabled])
 
+  const closeWindow = () => {
+    window.close()
+  }
+
   return (
     <FocusArea direction="horizontal">
       {innerProps => (
         <div
-          className={tailwindMerge.twMerge(
-            'pointer-events-auto flex shrink-0 cursor-default items-center gap-pages rounded-full px-page-switcher-x',
-            page === Page.editor && 'bg-frame backdrop-blur-default'
-          )}
+          className="pointer-events-auto flex h-12 shrink-0 grow cursor-default items-center rounded-full"
           {...innerProps}
         >
-          {PAGE_DATA.map(pageData => {
-            const error = ERRORS[pageData.page]
+          {SHOW_CLOSE_BUTTON && (
+            <div
+              className={tailwindMerge.twMerge(
+                'flex bg-primary/5 py-[18px] pl-[19px] pr-[29px]',
+                pageIndex === 0 && 'rounded-br-3xl'
+              )}
+            >
+              <ariaComponents.CloseButton onPress={closeWindow} />
```

Comment:
true. iirc we moved away from supporting the native macos menu a while back for some reason... i suppose there's no reason to keep the close buttons around, although maybe it's still useful to keep this element around purely for the padding in case we want to support macos traffic lights again later down the line?

Reply:
Even if we add macos traffic light buttons, we need to keep in mind that these paddings shall collapse in fullscreen mode(because the traffic light goes to the native topbar.

Example of incorrect behavior:

1. "Normal" mode: 
![CleanShot 2024-06-17 at 10 00 10](https://github.com/enso-org/enso/assets/61194245/a41e1ae0-3218-4e55-8b09-9af28b70807f)
2. Fullscreen mode:
![CleanShot 2024-06-17 at 10 00 51](https://github.com/enso-org/enso/assets/61194245/5b2dab5a-1539-4ba6-864e-50b8152e2c06)
![CleanShot 2024-06-17 at 10 01 35](https://github.com/enso-org/enso/assets/61194245/94386573-a517-4c73-9ba1-0a95962d8d0e)